File: TransCoder_tokenized_test_set_functions/java/MINIMUM_XOR_VALUE_PAIR_1.java
static int minXOR ( int arr [ ], int n ) {
  Arrays . parallelSort ( arr );
  int minXor = Integer . MAX_VALUE;
  int val = 0;
  for ( int i = 0;
  i < n - 1;
  i ++ ) {
    val = arr [ i ] ^ arr [ i + 1 ];
    minXor = Math . min ( minXor, val );
  }
  return minXor;
}

File: results/py/MINIMUM_XOR_VALUE_PAIR_1.cpp
int f_filled ( int * arr , int n ) {
  sort ( arr , arr + n ) ;
  int minXor = ( int ) INT_MAX ;
  int val = 0 ;
  for ( int i = 0 ;
  i <= n - 1 ;
  i ++ ) {
    val = arr [ i ] ^ arr [ i + 1 ] ;
    ;
    minXor = min ( minXor , val ) ;
  }
  ;
  return minXor ;
}
